src/header/css/header-menus.pcss

```diff
@@ -272,6 +350,9 @@
 		padding: 15px 35px 35px;
 
 		&[data-filled=""] {
+			&[data-has-structured="true"] {
+				width: calc(585px + 35px * 2);	
```

Comment:
what are these magical numbers from?

Reply:
@paulcpederson 
line #358 has an example of these magic numbers which were put in place by the previous developer.. I just took those and bumped them up to accommodate the two column system.

Reply:
ok, it's ExistingMagic™